cucumber.js

```diff
@@ -4,7 +4,6 @@ const config = {
     formatOptions: { snippetInterface: "synchronous" },
     requireModule: ["ts-node/register"],
     require: ["features/step_definitions/**/*.ts"],
-    publishQuiet: true,
```

Comment:
NOTE: The `publishQuiet` option is deprecated.
https://github.com/cucumber/cucumber-js/blob/main/docs/deprecations.md#publishquiet